subcommand_plugin_examples/c_subcommand_plugin_tutorial/README.md

```diff
@@ -0,0 +1,243 @@
+[template]: https://github.com/conda/conda-plugin-template/subcommand_plugin_examples/c_subcommand_plugin_tutorial/generate
+[pyproject.toml docs]: https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml
+[entrypoints docs]: https://packaging.python.org/en/latest/specifications/entry-points/
+[licenses]: https://docs.conda.io/projects/conda/en/latest/dev-guide/plugin-api/index.html#a-note-on-licensing
+[pep 621]: https://peps.python.org/pep-0621/
+[pluggy docs]: https://pluggy.readthedocs.io/en/stable/index.html
+
+# Custom conda Subcommand Plugin Tutorial Written in C
+
+In this tutorial, we will create a new conda subcommand written in C that converts Celsius to Fahrenheit.
+
+To follow along with this guide, make sure you have the latest conda and conda-build installed:
+
+```bash
+(base) $ conda update conda conda-build
+```
+
+## Project directory structure
+
+Set up your working directory and files as shown below (or create a new repo using this [template][template]):
```

Comment:
```suggestion
Set up your working directory and files as shown below (or create a new repository using this [template][template]):
```

Let's make sure to spell out things completely. Jargon like this can be a little off putting for some and hard to understand.